spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- To further explain the changes requested by both reviewers, for example, here, you've not added any test for the user, could you add some?